src/internal/connector/onedrive/collection.go

```diff
@@ -410,8 +410,13 @@ func (oc *Collection) populateItems(ctx context.Context, errs *fault.Bus) {
 
 					// check for errors following retries
 					if err != nil {
-						logger.Ctx(ctx).With("error", err.Error()).Error("downloading item")
+						if clues.HasLabel(err, graph.LabelsMalware) {
+							logger.Ctx(ctx).Infow("malware item", clues.InErr(err).Slice()...)
+						} else {
+							logger.Ctx(ctx).With("error", err.Error()).Error("downloading item")
+						}
 						el.AddRecoverable(clues.Stack(err).WithClues(ctx).Label(fault.LabelForceNoBackupCreation))
```

Comment:
do we want to only mark backups as not completing if they don't have malware? Otherwise we'll continue to see backup failures if we somehow miss some of the malware checks prior to attempting to download the item's data

Reply:
Yeah, fair catch.